internal/restic/lock_test.go

```diff
@@ -238,6 +239,7 @@ func TestLockRefresh(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	time.Sleep(time.Millisecond)
```

Comment:
Are we sleeping one millisecond here? Why?

Reply:
Sometimes, Go is too fast :)